### PR TITLE
Implementation of AddShips function

### PR DESCRIPTION
**Pull request by meghamittall**

### CODE REVIEW

File: battleship.py [hunk 1]
```diff
@@ -89,7 +115,17 @@ def checkShip(grid, ship):
 Returns: 2D list of ints
 '''
 def addShips(grid, numShips):
-    return
+    current = 0
+    while current < numShips:
+        ship = createShip()
+        bool = checkShip(grid,ship)
+        if bool == True:
```
Comment (Pragya28) [inline battleship.py:122]: No need of checking equality with True in case of boolean.

File: battleship.py [hunk 2]
```diff
@@ -89,7 +115,17 @@ def checkShip(grid, ship):
...
-    return
+    current = 0
+    while current < numShips:
+        ship = createShip()
+        bool = checkShip(grid,ship)
+        if bool == True:
+            for i in range(len(ship)):
+                x = ship[i][0]
+                y = ship[i][1]
+                grid[x][y] = 2
```
Comment (Pragya28) [inline battleship.py:126]: Use variable name instead of number

**Review by Pragya28 — CHANGES_REQUESTED** [review]

File: battleship.py (hunk 1 above)
Comment (meghamittall) [reply battleship.py]: @Pragya28 I made changes in the function. Now you can check.

File: battleship.py (hunk 2 above)
Comment (meghamittall) [reply battleship.py]: @Pragya28 I did the changes.

**Review by Pragya28 — COMMENTED** [review]
Good work @meghamittall.

**Pull request merged**